packages/back-end/src/services/GBCloudSDKService/middleware.ts

```diff
@@ -0,0 +1,108 @@
+import { NextFunction, Request, Response } from "express";
+import {
+  GrowthBook,
+  Attributes,
+  configureCache,
+  setPolyfills,
+} from "@growthbook/growthbook";
+import { AppFeatures } from "front-end/types/app-features";
+import { GROWTHBOOK_SECURE_ATTRIBUTE_SALT } from "shared/constants";
+import { getAccountPlan, getLicense } from "enterprise";
+import { IS_CLOUD } from "../../util/secrets";
+import { AuthRequest } from "../../types/AuthRequest";
+import { sha256 } from "../features";
+import { logger } from "../../util/logger";
+
+export async function initializeSdk(
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
+  setPolyfills({
+    fetch: require("node-fetch"),
+  });
+
+  const gb = new GrowthBook<AppFeatures>({
+    apiHost: "https://cdn.growthbook.io",
+    clientKey:
+      process.env.NODE_ENV === "production"
+        ? "sdk-ueFMOgZ2daLa0M"
+        : "sdk-UmQ03OkUDAu7Aox",
+  });
+  req.app.locals.growthbook = gb;
+
+  // Disable SSE
+  configureCache({ backgroundSync: false });
+
+  res.on("close", () => {
+    gb.destroy();
+  });
+
+  try {
+    await promiseTimeout(
+      // Do not subscribe individual SDK instances to streaming updates
+      gb.loadFeatures({ autoRefresh: false }),
```

Comment:
There's a built-in `timeout` prop.  Is there a reason we're not using that?

Reply:
I moved all this to https://github.com/growthbook/growthbook/pull/1505/files and addressed your feedback